complete.go

```diff
@@ -66,7 +66,7 @@ func (o *opCompleter) OnComplete() {
 	buf := o.op.buf
 	rs := buf.Runes()
 
-	if o.IsInCompleteMode() && runes.Equal(rs, o.candidateSource) {
+	if o.IsInCompleteMode() && len(rs) > 0 && runes.Equal(rs, o.candidateSource) {
```

Comment:
Use `runes.Equal(rs, o.candidateSource)` here is want to indicate whether user press twice `<tab>`.
`o.candidateSource` will be set to nil when `o.ExitCompleteSelectMode()` is called.
The problem is `runes.Equal([], nil)` is true.
So I suggests that change `len(rs)` to `o.candidateSource != nil`